docs/document/docs/components/desktop/tabs.md

```diff
@@ -0,0 +1,100 @@
+---
+id: tabs
+title: Tabs
+sidebar_label: Tabs
+---
+
+## Overview
+
+A multiple tabs component that can switch displaying contents with buttons.
+
+<div class="sample-container" id="tabs">
+  <div id="sample-container__components"></div>
+</div>
+<script src="/js/samples/desktop/tabs.js"></script>
+
+---
+
+## Specification
+
+### Property
+Here is a list of properties that can be used for modifying the component:
+
```

Comment:
I think this order will be better:
1. className
2. id
3. value
4. borderVisible
5. visible
6. items
7. Item.content
8. Item.label
9. Item.value
10. Item.disabled
11. Item.visible